inst/extdata/reportingVariables/Mapping_EuropeanEnergyDatasheets.csv

```diff
@@ -183,10 +191,14 @@
 "Energy Balance|Final non-energy consumption";"Mtoe";"FE (EJ/yr)";0.041868
 "Energy Balance|Final non-energy consumption";"Mtoe";"FE|Liquids (EJ/yr)";0.041868
 "Energy Balance|Final non-energy consumption";"Mtoe";"FE|Industry (EJ/yr)";0.041868
+"Energy Balance|Final non-energy consumption";"Mtoe";"FE|Non-energy Use|Industry (EJ/yr)";0.041868
 "Energy Balance|Final energy consumption";"Mtoe";"FE (EJ/yr)";0.041868
 "Energy Balance|Final energy consumption|Solid fossil fuels";"Mtoe";"FE|Solids (EJ/yr)";0.041868
```

Comment:
sorry, can you add two additional lines? 

"Energy Balance|Final energy consumption|Solid fossil fuels";"Mtoe";"FE|Solids|Fossil (EJ/yr)";0.041868
"Energy Balance|Final energy consumption|Solid fossil fuels";"Mtoe";"SE|Solids|Fossil (EJ/yr)";0.041868